src/documents/models.py

```diff
@@ -286,6 +286,10 @@ def find_renamed_document(self, subdirectory=""):
         # Go up in the directory hierarchy and try to delete all directories
```

Comment:
We actually don't try to delete anything, but just search for a file.

Reply:
fixed.